features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx

```diff
@@ -212,6 +213,18 @@ class Protected extends Component {
         if (user) {
             this.setState({ user });
             settingPromise.then((settingsNew) => this.setState({ settings: settingsNew }));
+            api.getTenantInformation(user.name)
+                .then((result) => {
+                    const { tenantDomain } = result.body;
+                    if (tenantDomain === 'carbon.super') {
+                        this.setState({ isTenant: false });
+                    } else {
+                        this.setState({ isTenant: true });
+                    }
+                })
+                .catch((error) => {
+                    console.log(error);
```

Comment:
Shall we alert the user about the error ? and log

Reply:
Fixed with `f01475d`